lib/src/signed_video_h26x_sign.c

```diff
@@ -578,32 +573,59 @@ signed_video_add_nalu_part_for_signing_with_timestamp(signed_video_t *self,
 }
 
 SignedVideoReturnCode
-signed_video_get_sei(signed_video_t *self, uint8_t *nalu_data, size_t *nalu_data_size)
+signed_video_get_sei(signed_video_t *self, uint8_t *sei, size_t *sei_size)
 {
 
-  if (!self || !nalu_data_size) return SV_INVALID_PARAMETER;
-  *nalu_data_size = 0;
+  if (!self || !sei_size) return SV_INVALID_PARAMETER;
+  *sei_size = 0;
   if (self->num_of_completed_seis < 1) {
-    DEBUG_LOG("No items in |nalus_to_prepend_list|");
+    DEBUG_LOG("There are no completed seis.");
     return SV_OK;
   }
-  if (!nalu_data) {
-    // Assign the NALU size to the provided pointers.
-    *nalu_data_size = self->sei_data_buffer[0].sei_size;
+  if (!sei) {
+    // Assign the SEI size to the provided pointers.
+    *sei_size = self->sei_data_buffer[0].completed_sei_size;
     return SV_OK;
   }
-  // Assign the NALU size and NALU data to the provided pointers.
-  *nalu_data_size = self->sei_data_buffer[0].sei_size;
-  memcpy(nalu_data, self->sei_data_buffer[0].payload, *nalu_data_size);
+  // Assign the SEI size and SEI data to the provided pointers.
+  *sei_size = self->sei_data_buffer[0].completed_sei_size;
+  memcpy(sei, self->sei_data_buffer[0].payload, *sei_size);
 
-  // Reset the fetched NALU information from the sei buffer.
+  // Reset the fetched SEI information from the sei buffer.
   signed_video_nalu_data_free(self->sei_data_buffer[0].payload);
 
   --(self->num_of_completed_seis);
   shift_sei_buffer_index(self);
   return SV_OK;
 }
 
+static SignedVideoReturnCode
+signed_video_get_lastest_sei(signed_video_t *self, uint8_t *sei, size_t *sei_size)
```

Comment:
Rename `get_latest_sei`
Only public APIs have the preface "signed_video".

This function does not work. How did you test it?
It will get you the oldest non-completed SEI and not the latest completed SEI. You will also lose the oldest SEI.